src/main/java/uk/gov/digital/ho/hocs/info/api/TeamService.java

```diff
@@ -206,6 +207,24 @@ public void addUsersToTeam(List<UUID> userUUIDs, UUID teamUUID) {
         log.info("Added users with UUIDs {} to team with UUID {}", userUuidList, team.getUuid().toString(), value(EVENT, USERS_ADDED_TO_TEAM));
     }
 
+    @Transactional
+    public void patchTeam(UUID teamUUID, PatchTeamDto patchTeamDto) {
+        log.debug("Patching team {}", teamUUID);
+        Team team = getTeam(teamUUID);
+
+        if (!Objects.isNull(patchTeamDto.getDisplayName())
+                && !patchTeamDto.getDisplayName().equals(team.getDisplayName())) {
+            this.updateTeamName(teamUUID, patchTeamDto.getDisplayName());
+        }
+
+        if (!Objects.isNull(patchTeamDto.getUnitUUID())) {
+            UUID newUnitUuid = UUID.fromString(patchTeamDto.getUnitUUID());
+            if(!team.getUnit().getUuid().equals(newUnitUuid)) {
+                moveToNewUnit(newUnitUuid, teamUUID);
```

Comment:
theres a mix of Uuid and UUID

Reply:
I've changed all the additions to use Uuid, as this appears to be the most commonly used variant for our variables in this repo and is more standard. This is something we should standardise across all of our repos, because it varies randomly across the project.